docs/RECIPES.md

```diff
@@ -3,14 +3,14 @@
 Below are detailed examples on how to leverage `preflight` in various
```

Comment:
Can we lose "leverage"? How about "run"? or "use"? Not business-speak. After all, when I see that I think the idea here is to set up a teeter-totter, put Preflight on one side, and then jump on the other side to send it flying off somewhere...

Reply:
changed `leverage` to `run`